components/MenuComponents/MenuTab.js

```diff
@@ -32,7 +32,7 @@ const MenuTab = styled(MenuTabContainer)`
   ${props =>
     props.underline &&
     css`
-      border-bottom: ${props.router.pathname == props.link ? `3px solid ${props.theme.orange}` : '2px solid #c4c4c4'};
+      border-bottom: ${props.router.pathname == props.link ? `3px solid #faa21b` : '2px solid #c4c4c4'};
```

Comment:
i tried, but beause the 3px solid ... is inside qoute marks it wont work because it's a string. Wont work outside neither because of the props the whole code is inside a ${} which makes the site believe that var() is a javascript function.